src/isomorphic/classic/element/__tests__/ReactElement-test.js

```diff
@@ -266,7 +266,11 @@ describe('ReactElement', () => {
       React.createElement(Wrapper)
     );
 
-    expect(element._owner.getPublicInstance()).toBe(instance);
+    if (typeof element._owner.tag === 'number') { // Fiber reconciler
```

Comment:
@gaearon Shouldn't we be checking for the ReactDOMFeatureFlag instead? We shouldn't track implementation details like this in the tests.